test/extended/cluster/cl.go

```diff
@@ -146,8 +146,11 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 				// deploymentName is buildName without the -1 suffix
 				deploymentName := buildName[:len(buildName)-2]
 				e2e.Logf("Waiting for deployment: %q", deploymentName)
-				err = exutil.WaitForDeploymentConfig(oc.KubeClient(), oc.AppsClient().Apps(), ns, deploymentName, 1, oc)
-				o.Expect(err).NotTo(o.HaveOccurred())
+				out, err := oc.Run("rollout status deployment/" + deploymentName).Output()
```

Comment:
it must be a nicer way to do a rollout, currently I didn't find a rollout api.
@stevekuznetsov we can do a rollout for any kind and not just for deployment ?! (e.g build, buildconfig, and so on.. )

Reply:
Builds don't result in an application rolled out and running so I am not quite sure I understand what you mean.

Reply:
@stevekuznetsov ok, good to know, I was quite sure that we can do a rollout for builds as well